cfme/tests/configure/test_register_appliance.py

```diff
@@ -181,6 +182,8 @@ def test_rh_registration(
     Bugzilla:
         1532201
     """
+    if GH(('ManageIQ/manageiq', 20390)).blocks:
```

Comment:
Why not include this in the meta blockers?

This needs to be skipped/blocked like this:
https://github.com/ManageIQ/integration_tests/blob/55a9d4ab7c2337e19d017a5a5590485638ee6702/cfme/tests/networks/test_provision_to_virtual_network.py#L44

Reply:
@mshriver If you mean to use `GH('ManageIQ/manageiq:20390')` then I believe those approaches are equivalent.

Looking at the results I see the skip did work fine. The failed `test_rh_updates` is not related to this PR.